js-core-1/week-1/lesson.md

```diff
@@ -122,104 +101,107 @@ In programming there are different _types of_ data. You've used one data type al
 Computers recognise strings as a sequence of characters but to humans, strings are simply lines of text.
 
 ```js
-var message = "This is a string";
+const message = "This is a string";
 ```
 
 Notice that strings are always wrapped **inside of quote marks**. We do this so that the computer knows when the string starts and ends.
 
 You can check that the data is a string by using the `typeof` operator:
 
 ```js
-var message = "This is a string";
-var messageType = typeof message;
+const message = "This is a string";
+const messageType = typeof message;
 
 console.log(messageType); // logs 'string'
 ```
 
-### Exercise
-
-* Write a program that logs a message and its type
-
-#### Expected result
+### Exercise (5 minutes)
 
-```sh
-This is a string
-string
-```
+1. Write a program that:
+* creates a variable called `colors` 
+* assigns colors "blue" and "yellow" as a comma separate string to `colors` 
+* logs the type `colors` using `typeof`.
+2. What is the `typeof` a number?
 
 ## String concatenation
 
-You can add two strings together using the plus operator (`+`):
+You can add two strings together using the plus operator (`+`) or *string interpolation*. String interpolation is a useful JavaScript feature that allows you to inject variables directly into a string:
 
 ```js
-var greetingStart = "Hello, my name is ";
-var name = "Daniel";
+// Here is an example using the plus operator to concat strings
+const greetingStart = "Hello, my name is ";
+const name = "Daniel";
 
-var greeting = greetingStart + name;
+const greeting = greetingStart + name;
 
 console.log(greeting); // Logs "Hello, my name is Daniel"
 ```
 
-### Exercise
-
-* Write a program that logs a message with a greeting and your name
+```js
+// Here is example using the String interpolation to concat strings
+const greetingStart = "Hello";
+const name = "Daniel";
 
-#### Expected result
+const greeting = `${greetingStart}, my name is ${name}`;
```

Comment:
how was this received? was it not a big jump going from `string1 + string2` to the very weird syntax of: `${greetingStart}, my name is ${name}\;`

Reply:
@sansaid